### PR TITLE
Add support for including disk usage information

### PR DESCRIPTION
**Pull request by bpbp-boop**

### CODE REVIEW

**Comment by tersmitten** [comment]
You changed the mode of the files (from `100644` to `100755`). Can you revert that?

File: defaults/main.yml [hunk 1]
```diff
@@ -29,5 +29,10 @@ snmpd_users:
 snmpd_sys_location: 'Unknown'
 snmpd_sys_contact: Root <root@localhost>
```
Comment (tersmitten) [inline defaults/main.yml]: Why not set this to `true`?

File: defaults/main.yml [hunk 2]
```diff
@@ -29,5 +29,10 @@ snmpd_users:
...
 
+snmpd_disks_include_all: false
+snmpd_disks_include_all_threshold: '10%'
```
Comment (tersmitten) [inline defaults/main.yml]: and make this `[]`?

**Comment by tersmitten** [comment]
This is a [handy resource](http://manpages.ubuntu.com/manpages/vivid/en/man5/snmpd.conf.5.html) for more information on the topic

**Comment by bpbp-boop** [comment]
I've reverted the accidental file mode change.

snmpd_disks_include_all is defaulting to false so the user can opt in to disk monitoring
snmpd_disks has the root volume at 10% and an example of how to use it. I can change the README to be more explicit and remove the default?

**Pull request merged**